src/commonTest/kotlin/com/github/michaelbull/result/RunTest.kt

```diff
@@ -0,0 +1,8 @@
+package com.github.michaelbull.result
+
+import kotlinx.coroutines.CoroutineScope
+
+/**
+ * Workaround to use suspending functions in unit tests
+ */
+expect fun runTest(block: suspend (scope : CoroutineScope) -> Unit)
```

Comment:
I think you want [`runBlockingTest`](https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-test/kotlinx.coroutines.test/run-blocking-test.html)

Reply:
yep see other comment, unfortunately thats only for jvm projects